src/momento/internal/aio/_scs_data_client.py

```diff
@@ -442,6 +457,88 @@ async def list_remove_value(
             return CacheListRemoveValue.Error(convert_error(e))
 
     # SET COLLECTION METHODS
+    async def set_add_elements(
+        self,
+        cache_name: TCacheName,
+        set_name: TSetName,
+        elements: TSetElementsInput,
+        ttl: CollectionTtl = CollectionTtl.from_cache_ttl(),
+    ) -> CacheSetAddElementsResponse:
+        try:
+            self._log_issuing_request("SetAddElements", {})
+            _validate_cache_name(cache_name)
+            _validate_set_name(set_name)
+
+            item_ttl = self._default_ttl if ttl.ttl is None else ttl.ttl
+            request = _SetUnionRequest()
+            request.set_name = _as_bytes(set_name, self.__UNSUPPORTED_SET_NAME_TYPE_MSG)
+            request.elements.extend(_set_as_bytes(elements, self.__UNSUPPORTED_SET_ELEMENTS_TYPE_MSG))
+            request.ttl_milliseconds = int(item_ttl.total_seconds() * 1000)
```

Comment:
We can refactor these using the method I introduced

Reply:
I put in #241 for that.